setup.cfg

```diff
@@ -128,3 +128,30 @@ exclude_lines =
     def __repr__
     def __str__
     def _ipython_display_
+
+[mypy]
+python_version = 3.9
+
+files=weldx
+ignore_missing_imports = True
+strict_optional = False
+pretty = True
+allow_redefinition = True
+show_error_context = True
+show_error_codes = True
+show_column_numbers = True
+
+[mypy-weldx.asdf.*]
+ignore_errors = True
+[mypy-weldx.tests.*]
+ignore_errors = True
+# remove after refactoring the geometry module
+[mypy-weldx.geometry.*]
+ignore_errors = True
```

Comment:
Tests are ignored. Asdf for now too. Don't know if we want to include it at some point. Geometry should be fixed when it has been refactored.